test/source/greeter.cpp

```diff
@@ -5,14 +5,14 @@
 #include <string>
 
 TEST_CASE("Greeter") {
-  using namespace greeter;
+    using namespace greeter;
```

Comment:
warning: do not use namespace using-directives; use using-declarations instead [google-build-using-namespace]

    using greeter::Greeter;
    using greeter::LanguageCode;